fastly/products/botmanagement/api.go

```diff
@@ -1,25 +1,30 @@
-package bot_management
+package botmanagement
 
 import (
 	"github.com/fastly/go-fastly/v9/fastly"
 	"github.com/fastly/go-fastly/v9/fastly/products"
 	"github.com/fastly/go-fastly/v9/internal/productcore"
 )
 
-const ProductID = "bot_management"
+const (
+	ProductID   = "bot_management"
+	ProductName = "Bot Management"
+)
+
+type EnableOutput = products.EnableOutput
```

Comment:
Maybe for future clarity, we could add an annotation like the following to all instances of this type of aliasing...

```suggestion
// EnableOutput distinguishes the product type from the general form.
type EnableOutput = products.EnableOutput
```

Reply:
This is a good idea, I'll do that now so I won't forget.